client/src/controllers/ActionController.ts

```diff
@@ -0,0 +1,42 @@
+import { Controller } from '@hotwired/stimulus';
+
+//  <button data-controller="w-action"
+//  data-w-action-csrf-token-value = '{{ csrf_token|escapejs }}'
+//  data-w-action-post-url-value = '{{ view.get_enable_url }}'
+//  data-w-action-key-name-value = 'csrfmiddlewaretoken'
+//  data-action="click->w-action#enableAction"
+//  type="submit" class="button no">Enable</button>
+
```

Comment:
This JSDoc is malformed and won't work - be sure to double check the expected syntax from other controllers.

Also, remember that you cannot have whitespace between the JSDoc and the code.